numberformat_proposed.html

```diff
@@ -721,15 +842,22 @@ <h1>Properties of Intl.NumberFormat Instances</h1>
 
     <ul>
       <li>[[Locale]] is a String value with the language tag of the locale whose localization is used for formatting.</li>
+      <li>[[DataLocale]] is a String value with the language tag of the nearest locale for which the browser has data to perform the formatting operation.  (TODO: is this precisely correct??)</li>
       <li>[[NumberingSystem]] is a String value with the "type" given in Unicode Technical Standard 35 for the numbering system used for formatting.</li>
-      <li>[[Style]] is one of the String values `"decimal"`, `"currency"`, or `"percent"`, identifying the number format style used.</li>
-      <li>[[Currency]] is a String value with the currency code identifying the currency to be used if formatting with the `"currency"` style. It is only used when [[Style]] has the value `"currency"`.</li>
-      <li>[[CurrencyDisplay]] is one of the String values `"code"`, `"symbol"`, or `"name"`, specifying whether to display the currency as an ISO 4217 alphabetic currency code, a localized currency symbol, or a localized currency name if formatting with the `"currency"` style. It is only used when [[Style]] has the value `"currency"`.</li>
+      <li>[[Style]] is one of the String values `"decimal"`, `"currency"`, `"percent"`, or `"unit"`. This value corresponds closely with the [[UnitType]] value.</li>
+      <li>[[UnitType]] is one of the String values `"none"`, `"currency"`, or `"measure-unit"`. The value `"measure-unit"` is used for both [[Style]] `"percent"` and `"unit"`.</li>
```

Comment:
Why is UnitType represented separately from Style? It seems like the only difference here is organizing "percent" as a kind of "measure-unit"--am I misunderstanding?

Reply:
OK, I can see how UnitType corresponds to what you have in your locale database, and Style corresponds to the options bag and resolvedOptions, but it still seems pretty redundant to me. I don't have an opinion whether it's stored in the object as Style or UnitType, but I'd prefer logic at the usage sites to translate from one to the other rather than redundantly storing the information.